src/components/m-datepicker/_template.js

```diff
@@ -2,40 +2,97 @@ import html from 'nanohtml';
 import {
   getAllLocaleMonthsArray,
   getSpecificYears,
-  getLocalWeekdayArray } from '../../js/date';
+  DEFAULT_NEW_YEARS,
+  getLocalWeekdayArray,
+} from '../../js/date';
+
+const TODAY = 'TODAY';
+
 // tolocaldatestring()
-/* eslint-disable */
+const getStartDate = (yearsRange, startYear = TODAY, startMonth = TODAY) => {
+  let { lowerEndYear, higherEndYear } = yearsRange;
+
+  const date = new Date();
+
+  if (!lowerEndYear) {
+    lowerEndYear = date.getFullYear();
+  }
+
+  date.setFullYear(lowerEndYear);
+
+  if (!higherEndYear) {
+    higherEndYear = date.getFullYear() + DEFAULT_NEW_YEARS;
+  }
+
+  if (startMonth !== TODAY) {
+    date.setMonth(startMonth);
+  }
+
+  if (startYear !== TODAY) {
+    date.setFullYear(startYear);
+  }
+
+  const year = date.getFullYear();
+
+  if (year < lowerEndYear) {
+    return {
+      year: lowerEndYear,
+      month: 0,
+    };
+  }
+
+  if (year > higherEndYear) {
+    return {
+      year: higherEndYear,
+      month: 11,
+    };
+  }
+
+  return {
+    year,
+    month: date.getMonth(),
+  };
+};
+
 export default ({
   classes,
   buttonOk,
   buttonCancel,
   locale = 'en-uk',
-  maxYears = 50,
-  futureYears = true,
-}) => html`
-  <article class="${classes} m-datepicker__article">
-      <div class="m-datepicker__dropdown">
-        <axa-dropdown class="m-datepicker__dropdown__month" size="sm" value="${new Date().getMonth()}" items="
-          ${JSON.stringify(getAllLocaleMonthsArray(locale).map((month, index) => {
-            return { name: month, url: '#', value: index };
-          }))}
-        ">
+  // value = 'iso',
+  startYear = TODAY,
+  startMonth = TODAY,
+  lowerEndYear,
+  higherEndYear,
+}) => {
+  const startDate = getStartDate({ lowerEndYear, higherEndYear }, startYear, startMonth);
+  const specificYears = getSpecificYears({ lowerEndYear, higherEndYear });
+  return html`
+    <article class="${classes} m-datepicker__article">
+      <div class="">
+        <axa-dropdown data-month="true" class="m-datepicker__dropdown m-datepicker__dropdown__month js-datepicker__dropdown__month"
```

Comment:
I would use the native dropdown on mobiles

Reply:
note will need to change dynamically based on `resize`/`reorientation`, device detection...

Reply:
i don't think they are design-compliant. But is a very good point. What if we create an issue and verify it with Design and skip the implementation of it in this PR?

@NoahLiechti can u please verify it?

Reply:
Fine with me, I think it's in general good practice to use native UIs on mobiles.
Actually we should also question to utilize the build in `<input type="date" />` and check how we can be compliant 🤔

Reply:
we spoke about the <input type="date" /> option as well and i will investigate it when we do the organism